libs/application/templates/mortgage-certificate/src/fields/SelectProperty/PropertyTypeSelectField/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from 'react'
+import { FieldBaseProps } from '@island.is/application/types'
+import { PropertyTypes } from '../../../lib/constants'
+import { useLocale } from '@island.is/localization'
+import { propertySearch } from '../../../lib/messages'
+import { Box } from '@island.is/island-ui/core'
+import { SelectController } from '@island.is/shared/form-fields'
+
+interface PropertyTypeProps {
+  setPropertyType: React.Dispatch<
+    React.SetStateAction<PropertyTypes | undefined>
+  >
+  propertyType: PropertyTypes | undefined
+}
+
+export const PropertyTypeSelectField: FC<
```

Comment:
Yes, but since it is all intertwined alongside PropertyTypeSearchField and CheckedProperties into SelectProperty I feel like it is makes more sense to have it custom.